.github/workflows/build_and_test.yaml

```diff
@@ -0,0 +1,41 @@
+name: Build and Test
+on:
+  push:
```

Comment:
lets limit CI to only run on `main` and not run when `.md` files or images are updated
```
  push:
    branches:
      - main
    paths-ignore:
      - "**/*.md"
      - "**/*.png"
  pull_request:
    branches:
      - main
    paths-ignore:
      - "**/*.md"
      - "**/*.png"
```

Reply:
I'll make the path change; re: branches, we'll have release branches that we want this to run on (i.e. we already have https://github.com/envoyproxy/gateway/tree/release-v0.1.0). Let's leave it running on all branches for now, if we find we have branches that we *need* it not to run on, we can always add later. CI minutes are free anyway :)

Reply:
+1